### ISSUE TITLE
New date generators: weekday, month.

### ISSUE THREAD

**Issue by mradionov** [Marak/faker.js#250]
```
faker.date.weekday(); // 'Wednesday'
faker.date.weekday({ short: true }); // 'Wed'
faker.date.month(); // 'September'
faker.date.month({ short: true }); // 'Sep'
```
I've recently had been faking some data and felt a need in these "date words", to get them one usually ends up including a library or manually creating arrays of all the weekdays and month names. I wonder if it's gonna be a good addition to a library and won't overload it.

**Comment by Marak** [comment]
@mradionov -

I think it's a pretty good idea!

I would probably create some new data definitions ( in the specific locale ) for date words. Create a few arrays of values such as month and day names. From there you can add a few methods to `date.js` file.

Will accept a pull request for this functionality.

**Comment by mradionov** [comment]
Thanks, @Marak, your suggestions are clear to me and I'd love to make a pull request. One more question: is it required for a contributor to provide translations for all locales when adding a new generator or at least default 'en' would be enough (+ maybe a few more which contributor is aware of) ?

**Comment by Marak** [comment]
Just contribute whichever languages you can. 

`en` is a good one to start with, as others can translate from there.

**Comment by mradionov** [comment]
This feature was released in 3.0.1 and I guess the issue can be closed. Thanks.
